rebasehelper/specfile.py

```diff
@@ -536,6 +536,9 @@ def write_updated_patches(self, patches, disable_inapplicable):
     # PACKAGE VERSION RELATED METHODS #
     ###################################
 
+    def get_NVR(self):
+        return '{}-{}-{}'.format(self.get_package_name(), self.get_full_version(), self.get_release_number())
```

Comment:
You should use `get_release()` instead of `get_release_number()`. Dist tag is part of Release, NVR always contains it (unless it's undefined, but that's special case).